project/Dependencies.scala

```diff
@@ -169,7 +169,9 @@ object Dependencies {
 
   val conseilCommonInclude: Seq[ModuleID] =
     concat(
+      config,
       logging,
+      pureConfig,
```

Comment:
I think these libs can be removed from downstream modules that included them, since they also depend on common